client/js/components/statement/publicStatementModal/StatementModal.vue

```diff
@@ -68,33 +68,19 @@
       <section
         v-show="step === 0"
         data-dp-validate="statementForm">
-        <div
-          v-if="loggedIn === false"
-          :class="prefixClass('c-statement__formhint flash-info u-mb-0_5')">
-          <i
-            aria-hidden="true"
-            :class="prefixClass('c-statement__hint-icon fa fa-lg fa-info-circle')" />
-          <span
-            :class="prefixClass('block u-ml')">
+          <dp-inline-notification
+            v-if="loggedIn === false"
+            type="info">
             <p v-cleanhtml="statementFormHintStatement" />
-            {{ Translator.trans('error.mandatoryfields') }}
-          </span>
-        </div>
+            <p v-cleanhtml="Translator.trans('statement.modal.step.write.privacy_policy')" />
+            <p>{{ Translator.trans('error.mandatoryfields') }}</p>
+          </dp-inline-notification>
 
-        <div
+        <dp-inline-notification
           v-show="dpValidate.statementForm === false"
-          id="statementFormErrors"
```

Comment:
The id, tabindex and aria-labelledby attrs are needed to [focus the notification on error](https://github.com/demos-europe/demosplan-core/blob/badd2f3f3ec0654f389ed0933364e08db0d67e83/client/js/components/statement/publicStatementModal/StatementModal.vue#L1467).